source/agora/network/DNSResolver.d

```diff
@@ -161,9 +160,7 @@ public abstract class DNSResolver
                 // Just pick the first record.
                 // TODO: What is expected when there's more than one record ?
                 // Should we use it as a fall back, round robin, something else?
-                const straddr = format("%d.%d.%d.%d",
-                    res.rdata.a[0] >> 24 & 0xFF, res.rdata.a[0] >> 16 & 0xFF,
-                    res.rdata.a[0] >>  8 & 0xFF, res.rdata.a[0] >>  0 & 0xFF);
+                const straddr = IPv4(res.rdata.a[0]).toString();
```

Comment:
`fromHost`?

Reply:
Oops

Reply:
I guess this shows the need for a disabled ctor :)

Reply:
I agreed too quickly, this is actually correct, or at the very least doing the same thing as before.
I suggest we merge this and take a look at whether it was doing the right thing in the first place ?